File: Length of Last Word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int count = 0;
        bool have_space = true;
        for (char& c : s) {
            if (c == ' ') have_space = true;
            else if (have_space) {
                count = 1;
                have_space = false;
            } else
                count++;
        }
        return count;
    }
};

File: Length of Last Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        count = 0
        have_space = True
        for c in s:
            if c == " ":
                have_space = True
            elif have_space:
                count = 1
                have_space = False
            else:
                count += 1
        return count

# class Solution:
#     def lengthOfLastWord(self, s: str) -> int:
#         return len(s.strip().split(" ")[-1])
